runtime/src/main.rs

```diff
@@ -39,7 +39,15 @@ pub extern "C" fn entry_point() -> ! {
     cprintln!("{}", BANNER);
     let mut drivers = unsafe {
         Drivers::new_from_registers().unwrap_or_else(|e| {
-            caliptra_common::report_handoff_error_and_halt("Runtime can't load drivers", e.into())
+            // treat global exception as a fatal error
+            if e == CaliptraError::RUNTIME_GLOBAL_EXCEPTION {
+                handle_fatal_error(e.into());
```

Comment:
report_handoff_error_and_halt is a non-fatal error while handle_fatal_error is a fatal error and zeroizes the crypto peripherals. We get a RUNTIME_GLOBAL_EXCEPTION if disabling attestation on a non-fatal error fails, so then we should put RT in an unusable state.

Reply:
I guess it doesn't matter that much since report_handoff_error_and_halt infinitely loops, so caliptra is unusable anyway until reboot. Let me know if you think I should remove this change.

Reply:
I'm fine with leaving this as you have it here. I'm wondering if it makes sense to just always report a fatal error since it's technically fatal anyway. But we can do that in a follow-up.